api-reference/lease/find-leases-for-tenant.mdx

```diff
@@ -6,12 +6,25 @@ description: "This endpoint finds leases for a specific tenant based on provided
 
 The endpoint will automatically match the tenant information provided with an existing tenant within the system.
 
+At least one of the following tenant identifiers need to be provided:
+- Internal tenant identifier (`identifier`)
+- External tenant identifier (`external_ref`) and the group id the tenant belongs to (`group`)
```

Comment:
Open question, is 'external_ref' going to be confusing for our clients? It's external to us, but internal to them. What is this kind of ID in other APIs?

Reply:
I will leave this as external_ref for now as we have the explanation in the glossary, but if our clients feed back that they've found this confusing we can always change it.